package.json

```diff
@@ -32,6 +32,27 @@
     "ext": "js",
     "exec": "babel-node ./src/server/"
   },
+  "eslintConfig": {
+    "extends": [
```

Comment:
The extends block is taken from https://medium.com/@joelmasters/setting-up-eslint-for-nextjs-37163d4cabaa.